### DISCUSSION TITLE
帕姆任务好像可以每日多次，有考虑做不

### DISCUSSION THREAD

**Discussion by foodtooth** [LmeSzinc/StarRailCopilot#32] [Ideas]
b站有人总结帕姆任务。给信用点和各种材料，好像还是每日多次的

**Comment by BeatSeat** [comment]
好像一开始是多次刷新，后面好像是每天只有一次了，我记得的有：
1. 在列车上的，比如清扫地面、量身高
2. 在黑塔空间站的资源补充协议
3. 去雅利洛请教管理经验
总之我最近做到的就只有这些了
